src/vivarium_census_prl_synth_pop/tools/make_results.py

```diff
@@ -396,66 +393,33 @@ def perform_post_processing(
             obs_dir = build_output_dir(final_output_dir, subdir=observer)
             seed_ext = f"_{seed}" if seed != "" else ""
             write_shard_metadata(observer, obs_data, obs_dir, seed_ext)
-            write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.{extension}")
+            write_to_disk(obs_data.copy(), obs_dir / f"{observer}{seed_ext}.parquet")
 
 
 def load_data(raw_results_dir: Path, seed: str) -> Dict[str, pd.DataFrame]:
-    # Loads in all observer outputs and stores them in a dict with observer name as keys.
+    """Loads in all observer outputs and stores them in a dict with observer name as keys."""
     observers_results = {}
     for observer in FINAL_OBSERVERS:
-        obs_dir = raw_results_dir / observer
-        if seed == "":
```

Comment:
I'm able to delete this seed logic by instead adding `seed` as an arg to `read_datafile` with a defautl of "" and handle it in there